pysen/pyproject_model.py

```diff
@@ -53,7 +53,7 @@ class PluginConfig:
 
 @dataclasses.dataclass
 class Config:
-    version: Optional[VersionRepresentation] = None
+    version: Optional[str] = None
```

Comment:
[RFC] I'm wondering we can simply change `version` in `Config` and `py_version` in `MypyModuleOption` to str. It makes config file consistent to their dataclass attributes. I think we can hide `VersionRepresentation` and `PythonVersion` internally.

https://github.com/pfnet/pysen/compare/d00e7f4..52737ff

Reply:
OK!
Thank you for the suggestion!